internal/vault/client.go

```diff
@@ -537,14 +553,14 @@ func (c *defaultClient) renew(ctx context.Context) error {
 		return errs
 	}
 
-	resp, err := c.Write(ctx, "/auth/token/renew-self", nil)
+	resp, err := c.Write(ctx, nil)
```

Comment:
Does this still work without "/auth/token/renew-self" in the request?

Reply:
Oh, good catch. Bad refactoring.